include/shell.h

```diff
@@ -90,20 +90,16 @@ class ShellHistory {
 public:
 	std::vector<std::string> GetCommands(uint16_t code_page) const;
 	void Append(std::string command, uint16_t code_page);
-
-	ShellHistory();
-	~ShellHistory();
-	ShellHistory(const ShellHistory&)            = delete;
-	ShellHistory& operator=(const ShellHistory&) = delete;
-	ShellHistory(ShellHistory&&)                 = delete;
-	ShellHistory& operator=(ShellHistory&&)      = delete;
+	void SetPathFromConfig();
```

Comment:
Good riddance. Could mark the class `final` as well (`class ShellHistory final`).